src/index.ts

```diff
@@ -28,27 +28,19 @@ import { KolibriCli } from './kolibri-cli';
 
 async function main() {
     try {
-        const argv = await yargs(hideBin(process.argv)).argv;
-        let path: string = `${homedir()}/.kolibri-cli/config.env`;
-        createDefaultConfigFileIfNotExists(path);
+        const defaultConfigFile: string = `${homedir()}/.kolibri-cli/config.env`;
+        createDefaultConfigFileIfNotExists(defaultConfigFile);
```

Comment:
A non default path will not be created as it is now. I think this should be below the line  const configFile = await parseConfigFileArg() ?? defaultConfigFile;
And maybe name createConfigFileIfNotExists.

const configFile = await parseConfigFileArg() ?? defaultConfigFile;
createConfigFileIfNotExists(configFile)

Reply:
aah yes, nvm then